pkg/web/web.go

```diff
@@ -168,16 +172,35 @@ func New(ctx context.Context, opts ...Option) (*Server, error) {
 		server: server,
 	}
 
-	var staticDir http.Dir
+	var staticPath string
 	for _, path := range options.staticSearchPaths {
 		if s, err := os.Stat(path); err == nil && s.IsDir() {
-			staticDir = http.Dir(path)
+			staticPath = path
 			break
 		}
 	}
-	if staticDir != "" {
-		logger.WithFields(log.Fields("path", staticDir, "mount", options.staticMount)).Debug("Serving static assets")
+	if staticPath != "" {
+		staticDir := http.Dir(staticPath)
+		logger := logger.WithFields(log.Fields("path", staticDir, "mount", options.staticMount))
+		logger.Debug("Serving static assets")
 		s.Static(options.staticMount, staticDir, middleware.Immutable)
```

Comment:
unrelated to this PR, but let's remove this line while we're editing this file

Reply:
Removed immutable middleware